force-app/main/default/classes/BDI_DataImportService.cls

```diff
@@ -346,6 +346,36 @@ global with sharing class BDI_DataImportService {
         }
     }
 
+    /*******************************************************************************************************
+    * @description Instrument that multiple gift were successfully processed through the DataImport Batch process
+    * @param Id Apex Batch Job to find related DataImport__c records for
+    * @return void
+    */
+    public static void instrumentGiftEntryUIResults(Id apexJobId) {
```

Comment:
I think that either this function or the overload could get renamed. Does `instrumentGiftEntryBatchResults` or `instrumentGiftCount` feel like they'd work better?